al_servicec.js

```diff
@@ -226,29 +226,10 @@ class AzcollectC extends AlServiceC {
                 this._collectorType = collectorType;
                 break;
             default:
-                // Keep for backward compatibility
-                this._collectorType = COLLECTOR_TYPES.CWE;
-                break;
-                // TODO: Should have exception here when cwe-collector is migrated
-                // throw `Unknown collector type: ${collectorType}`;
+                throw `Unknown collector type: ${collectorType}`;
         }
     }
 
-    doCheckin(checkinValues) {
```

Comment:
@kkuzmin we will merge this one only after we're ready to migrate GD, right?

Then I suggest to rename it to "[DO NOT MERGE]" and describe why in the description, so everybody can see it's in "waiting" state.